modules/ROOT/pages/index.adoc

```diff
@@ -3,17 +3,17 @@
+OpenShift Pipelines project is based on the upstream project link:https://github.com/tektoncd[Tekton] and is available as a Developer Preview release. Developer Preview releases have features and functionality that might not be fully tested. Customers are encouraged to use and provide feedback on Developer Preview releases. Red Hat does not commit to fixing any reported issues and the provided features may not be available in future releases.
 
 Developer Preview releases are not production ready and customers are advised to avoid running the software for production or business-critical workloads. Limited documentation is available for Developer Preview releases. Customers are NOT able to open support cases against the Developer Preview releases.
 
-If you need assistance with OpenShift Pipelines, please reach out on the following mailing list where a member of the Red Hat Development Team will be happy to assist you based on availability and work schedules: pipelines-interest@redhat.com
+If you need assistance with OpenShift Pipelines, reach out on the following mailing list where a member of the Red Hat Development Team will be happy to assist you based on availability and work schedules: pipelines-interest@redhat.com
 ====
 
-OpenShift Pipelines are cloud-native, continuous integration and delivery solutions based on Kubernetes resources. It uses link:https://tekton.dev[Tekton] Pipelines to automate deployments across multiple platforms by abstracting away the underlying details. Tekton introduces a number of standard link:https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/[Custom Resource Definitions (CRDs)] for defining pipelines that are portable across Kubernetes distributions.
+OpenShift Pipelines are cloud-native, continuous integration and continuous delivery (CICD) solutions based on Kubernetes resources. It uses link:https://tekton.dev[Tekton] Pipelines to automate deployments across multiple platforms by abstracting away the underlying details. Tekton introduces a number of standard link:https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/[Custom Resource Definitions (CRDs)] for defining pipelines that are portable across Kubernetes distributions.
 
 * OpenShift Pipelines are designed for microservices and decentralized teams.
-* OpenShift Pipelines uses standard CI/CD pipeline definitions that are easy to extend and integrate with existing Kubernetes tools, enabling you to scale on-demand.
+* OpenShift Pipelines uses standard CICD pipeline definitions that are easy to extend and integrate with existing Kubernetes tools, enabling you to scale on-demand.
```

Comment:
why is CICD better than CI/CD?

Reply:
Both CI/CD and CICD are acceptable terms ( as searched on google). However, I also now went through the other Red Hat  docs, and they use Ci/CD. Hence, updated it back to CI/CD